wdio.conf.js

```diff
@@ -8,4 +8,10 @@ if (process.env.npm_package_name !== 'terra-clinical') {
   wdioConfig.specs = [path.join(__dirname, 'packages', process.env.npm_package_name, 'tests', 'wdio', '**', '*-spec.js')];
 }
 
+const travis = process.env.TRAVIS;
+
+if (travis) {
+  wdioConfig.host = 'localhost';
```

Comment:
Does this configuration when using the selenium -docker resolve the error you were seeing and the need to use the standalone-chrome-travis service?

Reply:
This lets us call the stand Alone chrome docker instance I spin up per executor. So we still need it and aren’t using the selenium docker service. Seems like it was having trouble promptly starting.